tasks/main.yml

```diff
@@ -0,0 +1,54 @@
+---
+- name: install cron (bionic support)
+  apt: name={{ item }} update_cache=yes state=present
+  with_items:
+    - cron
+
+- name: Check ci_inventory is defined
```

Comment:
This assertion should be at the top. The intention is to sanity check the role before it starts any work.

Reply:
Updated in the latest commit.